en/components/tree.md

```diff
@@ -0,0 +1,313 @@
+            {{ child.text }}
+		</igx-tree-node>
+	</igx-tree-node>
+</igx-tree>
+```
+```typescript
+export class MyTreeViewComponent {
+  public data: { [key: string]: any, valueKey: string } = MY_DATA;
+  @ViewChild(IgxTreeComponent, { read: IgxTreeComponent })
+  public tree;
+
+  findNode(valueKey: string) {
+    const comparer: (nodeData: any, node: IgxTreeNodeComponent) => boolean =
+      (data: any, node: IgxTreeNodeComponent) => nodeData.valueKey === data;
+    this.tree.findData(valueKey, comparer);
+  }
+}
+```
+### Templating
+To create a reusable template for your nodes, declare `<ng-template>` **within `igx-tree`**. 
+```html
+ <igx-tree>
+    <igx-tree-node #myNode *ngFor="let node of data" [data]="node">
+        <ng-template *ngTemplateOutlet="nodeTemplate; context { $implicit: myNode }">
+        </ng-template>
+        <igx-tree-node *ngFor="let child of node.ChildCompanies" [data]="child">
+            {{ child.CompanyName }}                           
+        </igx-tree-node>
+    </igx-tree-node>                      
+                   
+    <ng-template #nodeTemplate let-node>
+        <igx-combo type="line" [(ngModel)]="node.CompanyName" [displayKey]="'CompanyName'" [data]="data" width="220px"></igx-combo>
+    </ng-template>
+</igx-tree>
+```
+Additionally, by using the [expandIndicator]({environment:angularApiUrl}/classes/igxtreecomponent.html#expandIndicator) input you have the ability to set a custom template to be used for rendering the expand/collapse indicators of nodes.
+```html
+<igx-tree>
+    <igx-tree-node *ngFor="let node of data" [data]="node">
+    </igx-tree-node>
+    <ng-template igxTreeExpandIndicator let-expanded>
+        <igx-icon>{{ expanded ? 'expand_less' : 'expand_more' }}</igx-icon>
+    </ng-template>
+</igx-tree>
+```
+
+## Angular Tree Selection
+In order to setup node selection in the `igx-tree`, you just need to set its [selection]({environment:angularApiUrl}/classes/igxtreecomponent.html#selection) property. This property accepts the following three modes: **None**, **Bi-State** and **Cascading**. Below we will take a look at each of them in more detail.
+### None
+In the `igx-tree` by default node selection is disabled. Users cannot select or deselect a row through UI interaction, but these actions can still be completed through the provided API method.
+### Bi-State
+To enable bi-state node selection in the `igx-tree` just set the [selection]({environment:angularApiUrl}/classes/igxtreecomponent.html#selection) property to **BiState**. This will render a checkbox for every node.  Each node has two states - selected or not.  This mode supports multiple selection.
+```html
+<igx-tree [selection]="'BiState'">
+</igx-tree>
+```
+### Cascading
+To enable cascade node selection in the `igx-tree` just set the selection property to **Cascading**. This will render a checkbox for every node. 
```

Comment:
```suggestion
To enable cascade node selection in the `igx-tree`, just set the selection property to **Cascading**. This will render a checkbox for every node. 
```